pkg/cnab/provider/credentials.go

```diff
@@ -26,7 +26,7 @@ func (r *Runtime) loadCredentials(b cnab.ExtendedBundle, args ActionArguments) (
 		var cset credentials.CredentialSet
 		var err error
 		if r.isPathy(name) {
-			cset, err = r.loadCredentialFromFile(name)
+			return nil, errors.Errorf("cannot use file path %s as credential set source", name)
```

Comment:
What if we remove this check (and isPathy) entirely? Then if someone tries to pass a file, the error will be from when we tried to load the credential set, e.g. "credential set ./mycreds.json not found".

As we have been making breaking changes for v1, I have been removing deprecation warnings, shims and checks for old behavior that was only supported in v0.* versions. The v1 release is our one chance to make a correct previous decisions and simplify the code, before we _have_ to keep around a bunch of code for backwards compatibility. 😀

Reply:
Sounds good. That makes sense, Thanks for explaining the context with me!

Reply:
Would you please make the same change for parameters as well so they are consistent?